selenium_base.py

```diff
@@ -0,0 +1,68 @@
+from selenium import webdriver
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.select import Select
+from users.models import CustomUser
+
+webdriver = webdriver.Firefox
+
+class SeleniumTestsBase(StaticLiveServerTestCase):
+    fixtures = [ 'institution/fixtures/institutions.yaml', 'project/fixtures/funding_sources.yaml', ]
+
+    serialized_rollback = True
+
+    def get_url(self, url):
+        self.selenium.get(self.live_server_url+url)
+
+    def fill_form_by_id(self, fields):
+        for field, value in fields.items():
+            element = self.selenium.find_element_by_id(field)
+            element.send_keys(value)
+
+    def click_by_text(self, text):
+        button = self.selenium.find_element_by_xpath("//*[text()='"+text+"']")
+        button.click()
+
+    def select_from_dropdown(self, id, index):
+        element = Select(self.selenium.find_element_by_id(id))
+        element.select_by_index(index)
+
+    def sign_in(self):
+        """
+        Sign in as a preexisting test user
+        """
+        # Sign in using the admin page
```

Comment:
/external/login/ is the login form for external collaborators. /admin is a the login form for admins. I will update the comment.